docs/hazmat/backends/openssl.rst

```diff
@@ -93,7 +93,8 @@ OS random sources
 -----------------
 
 On macOS and FreeBSD ``/dev/urandom`` is an alias for ``/dev/random`` and
-utilizes the `Yarrow`_ algorithm.
+utilizes the `Yarrow`_ algorithm. FreeBSD version 10 and higher use the
+`Fortuna`_ algorithm.
```

Comment:
According to our docs, we only support FreeBSD 11 and up, so let's simplify this to saying macOS is Yarrow and FreeBSD is Fortuna